File: client/src/main/java/au/com/codeka/warworlds/game/starfield/scene/SelectionIndicatorSceneObject.java
package au.com.codeka.warworlds.game.starfield.scene;

import au.com.codeka.common.Colour;
import au.com.codeka.warworlds.opengl.DimensionResolver;
import au.com.codeka.warworlds.opengl.SceneObject;

/**
 * A {@link SceneObject} that we can add to a star or fleet to indicate that it's the currently-
 * selected one.
 */
public class SelectionIndicatorSceneObject extends BaseIndicatorSceneObject {
  public SelectionIndicatorSceneObject(DimensionResolver dimensionResolver) {
    super(dimensionResolver, Colour.WHITE, 5.0f);
  }
}

File: client/src/main/kotlin/au/com/codeka/warworlds/client/game/starfield/scene/SelectionIndicatorSceneObject.kt
package au.com.codeka.warworlds.client.game.starfield.scene

import au.com.codeka.warworlds.client.game.starfield.scene.BaseIndicatorSceneObject
import au.com.codeka.warworlds.client.opengl.DimensionResolver
import au.com.codeka.warworlds.client.opengl.SceneObject
import au.com.codeka.warworlds.common.Colour

/**
 * A [SceneObject] that we can add to a star or fleet to indicate that it's the currently-
 * selected one.
 */
class SelectionIndicatorSceneObject(dimensionResolver: DimensionResolver)
  : BaseIndicatorSceneObject(dimensionResolver, "SelectionIndicator", Colour.WHITE, 5.0f)
